### PR TITLE
added ecr.tf - removed unneeded variables.tf

### PR DESCRIPTION
**Pull request by sgsollie**

### CODE REVIEW

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/check-my-diary-dev/resources/ecr.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
```
Comment (digitalronin) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/check-my-diary-dev/resources/ecr.tf]: The 'backend' and 'provider' are also specified in your `main.tf` - this is causing terraform to fail.

Comment (sgsollie) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/check-my-diary-dev/resources/ecr.tf]: amended

**Comment by sgsollie** [comment]
no visibility on concourse-ci job output - guessing that is down to permissions

**Comment by digitalronin** [comment]
This is the error on concourse:

Error: kubernetes_secret.checkmydiary_ecr_credentials: metadata.0.name a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')

**Comment by digitalronin** [comment]
I think the problem is you've got this;
```
    access_key_id     = "${module.checkmydiary-service-dev.access_key_id}"
```
instead of this;
```
    access_key_id     = "${module.checkmydiary_service_dev.access_key_id}"
```
etc.

**Comment by sgsollie** [comment]
It looks to me line 9, couldn't have underscores. Have pushed changes.

**Pull request merged**